shakedown/dcos/helpers.py

```diff
@@ -29,10 +29,15 @@ def get_transport(host, username, key):
         transport_master = start_transport(transport_master, username, key)
 
         if not transport_master.is_authenticated():
-           print('error: unable to authenticate ' + username + '@' + shakedown.master_ip() + ' with key ' + key_path)
-           return False
+            print('error: unable to authenticate ' + username + '@' + shakedown.master_ip() + ' with key ' + key_path)
+            return False
+
+        try:
+            channel = transport_master.open_channel('direct-tcpip', (host, 22), ('127.0.0.1', 0))
+        except paramiko.SSHException:
+            print('error: unable to connect to ' + host)
+            return False
```

Comment:
Should this be `False` or `None`? `False` does not match the defined return type in the method docs.